src/psyclone/tests/psyir/frontend/fortran_test.py

```diff
@@ -31,7 +31,8 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
-# Authors S. Siso, STFC Daresbury Lab
+# Author S. Siso, STFC Daresbury Lab
+# Modified by R. W. Ford, STFC Daresbury Lab
```

Comment:
pylint complains the "import six" is not needed in this file

Reply:
Sorry again.